module/VuFind/src/VuFind/Record/Loader.php

```diff
@@ -207,9 +210,9 @@ public function loadBatchForSource(
         $genuineRecords = [];
         if ($list->hasUnchecked()) {
             try {
+                $command = new RetrieveBatchCommand($source, $list->getUnchecked(), $params);
```

Comment:
The build failure is caused by this line being longer than the 86-character limit. Try changing it to:

```
                $command = new RetrieveBatchCommand(
                    $source,
                    $list->getUnchecked(),
                    $params
                );
```